packages/env/shared.ts

```diff
@@ -89,6 +89,7 @@ export const sharedEnvs = createEnv({
 		ELASTICSEARCH_PASSWORD: z.string(),
 		ELASTICSEARCH_USERNAME: z.string(),
 		ELASTICSEARCH_MESSAGE_INDEX: z.string(),
+		ELASTIC_DISABLED: z.string().optional(),
```

Comment:
Could you remove this and I'll fix it in a different PR

Reply:
Done! 👍